Extensions/Effects/crt-pixi-filter.js

```diff
@@ -0,0 +1,13 @@
+gdjs.PixiFiltersTools.registerFilterCreator('Crt', {
+  makePIXIFilter: function() {
+    var crt = new PIXI.filters.CRTFilter();
+
+    return crt;
+  },
+  updateParameter: function(filter, parameterName, value) {
+    if (!['curvature', 'lineWidth', 'lineContrast', 'verticalLine', 'noise', 'noiseSize', 'vignetting', 'vignettingAlpha', 'vignettingBlur', 'seed', 'time'].includes(parameterName)) return;
+
+    if (filter !== 'verticalLine') filter[parameterName] = gdjs.PixiFiltersTools.clampValue(value, 0, 20);
+    else filter.verticalLine = value === 1;
```

Comment:
This looks like a boolean? Put a `//TODO` so that we don't merge this while booleans are not supported in the IDE and this is not updated to show a boolean in the resources :)

Reply:
I can remove the ability to enable it instead, so we can still merge it? I love this filter and doubt anyone would want to enable the verticalLine feature

Reply:
Yep it's fine to go ahead without this and add this later :)